infra.bs

```diff
@@ -16,6 +16,20 @@ Abstract: The Infra Standard aims to define the fundamental concepts upon which
 Boilerplate: omit conformance, omit feedback-header, omit idl-index
 </pre>
 
+<style>
+#base64-table {
+  white-space: nowrap;
+  font-size: 0.6em;
+  column-width: 6em;
+  column-count: 5;
+  column-gap: 1em;
```

Comment:
Sad that this doesn't work in Firefox, ugh. I tried Googling and messing with stuff but no luck.